cg/meta/workflow/microsalt.py

```diff
@@ -103,7 +100,7 @@ def fastq_header(line):
 
         return rs
 
-    def link_sample(self, fastq_handler: BaseFastqHandler, sample: str, case: str):
+    def link_sample(self, fastq_handler: FastqHandler, sample: str, ticket: int):
```

Comment:
Returntype. `None` is also an important return type

Reply:
added